src/CoreBundle/private/library/classes/dbobjects/TCMSTableConf.class.php

```diff
@@ -195,6 +214,36 @@ public function &GetFields(&$oTableRow, $loadDefaults = false, $bDoNotUseAutoObj
         return $oFields;
     }
 
+    /**
+     * @param TdbCmsFieldConf|TCMSFieldDefinition $fieldDefinition
+     * @param array                               $sqlData
+     * @param LanguageServiceInterface            $languageService
+     * @return mixed|null
+     */
+    private function getDataForCurrentLanguage($fieldDefinition, array $sqlData, LanguageServiceInterface $languageService)
```

Comment:
The language is the same for every call, as it only relies on $this. So we should retrieve the language in the calling method and pass it instead of the LanguageService.

Reply:
But it might be null - so every caller would need to handle that case.

Reply:
Then I'd prefer retrieving the LanguageService also within the method. There will be a minimal performance hit, but having the callers retrieve and pass it, looks a bit odd.

Reply:
ok